### PR TITLE
[SDK-3692] Update readme to match new design

### PR DESCRIPTION
**Pull request by ewanharris**
### Description

Updates the project readme to match the new standard

- Updates readme structure
- Splits out contributing guide to a dedicated document

Still needs the new header image adding

### Checklist

- [x] I have added documentation for new/changed functionality in this PR or in auth0.com/docs
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used, if not the default branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,35 +2,33 @@
...
-- Our [Express Quickstart](https://auth0.com/docs/quickstart/webapp/express) is the quickest way to get up and running from scratch.
-- Use the [Examples for common configurations](https://github.com/auth0/express-openid-connect/blob/master/EXAMPLES.md) for use cases beyond the basics.
-- The [API documentation](https://auth0.github.io/express-openid-connect) details all configuration options, methods, and data that this library provides.
-- You can [run the sample application](https://github.com/auth0-samples/auth0-express-webapp-sample/tree/master) to see how this SDK functions without writing your own integration.
+- [Quickstart](https://auth0.com/docs/quickstart/webapp/express) - our guide for quickly adding Auth0 to your Express app.
+- [Sample](https://github.com/auth0-samples/auth0-express-webapp-sample/tree/master/01-Login) - an Express app integrated with Auth0.
+- [FAQs](./FAQ.md) - Frequently asked questions about express-openid-connect.
+- [Docs Site](https://auth0.com/docs) - explore our docs site and learn more about Auth0.
+- [Docs Site(https)
+
```
Comment (frederikprijck) [inline README.md:20]: Looks like a left-over.

```suggestion

```

File: README.md [hunk 2]
```diff
@@ -2,35 +2,33 @@
...
+## Getting Started
+
+### Requirements
+
+This library supports the following tooling versions:
+
+* Node.js `^10.19.0 || >=12.0.0`
```
Comment (frederikprijck) [inline README.md:27]: Both 10 and 12 are not in LTS anymore, even though we might be compatible, would it make sense to update this to mention the current LTS versions, assuming we support these?

Doesn't have to be in this PR, I just noticed it and wanted to raise it.

File: README.md [hunk 3]
```diff
@@ -2,35 +2,33 @@
...
 
 ## Install
 
-Node.js version **>=12.0.0** is recommended, but **^10.19.0** lts/dubnium is also supported.
+Using [npm](https://npmjs.org) in your project directory run the following command:
```
Comment (frederikprijck) [inline README.md:31]:
```suggestion
Using [npm](https://npmjs.org) in your project directory, run the following command:
```

File: README.md [hunk 4]
```diff
@@ -40,6 +38,28 @@ npm install express-openid-connect
 
 Follow our [Secure Local Development guide](https://auth0.com/docs/libraries/secure-local-development) to ensure that applications using this library are running over secure channels (HTTPS URLs). Applications using this library without HTTPS may experience "invalid state" errors.
 
+### Configure Auth0
+
+Create a **Regular Web Application** in the [Auth0 Dashboard](https://manage.auth0.com/#/applications).
+
+> **If you're using an existing application**, verify that you have configured the following settings in your Single Page Application:
```
Comment (frederikprijck) [inline README.md:45]: It looks like this is copy pasted from a SPA, we should remove the reference to SPA but also ensure the configuration below is correct for a Regular Web App, which might be different from a SPA.

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ewanharris) [reply README.md:27]: Yeah it didn't feel right having 10 and 12 here, but given that we only run tests against 12 currently I didn't want to add 14/16/18 without having at least one of them in CI

File: README.md (hunk 4 above)
Comment (ewanharris) [reply README.md:45]: I'd verified this using the sample, just overlooked the SPA reference here, will fix

File: README.md (hunk 2 above)
Comment (frederikprijck) [reply README.md:25]: Fair point, let's leave this for now.

Comment (adamjmcgrath) [reply README.md]: Yeah, that's out of date - will fix that

File: README.md [hunk 5]
```diff
@@ -40,6 +35,28 @@ npm install express-openid-connect
...
```
Comment (frederikprijck) [inline README.md:42]:
```suggestion
> **If you're using an existing application**, verify that you have configured the following settings in your Regular Web Application:
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**